src/main/java/net/minecraftforge/registries/HolderSetType.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.registries;
+
+import com.mojang.serialization.Codec;
+
+import net.minecraft.core.Holder;
+import net.minecraft.core.Registry;
+import net.minecraft.resources.ResourceKey;
+
+public interface HolderSetType
```

Comment:
The builtin Forge implementations of this interface seem to use this as a functional interface. Would there be a case where this interface would not be implemented as such (i.e. as a lambda or method reference)? If not, might be good to mark this as a `@FunctionalInterface`. If yes, then it should be documented that it can be implemented as a method reference.

Also, Forge naming convention calls for all interfaces to be prefixed with `I`.

```suggestion
@FunctionalInterface
public interface IHolderSetType
```